atlassian_jwt_auth/contrib/tests/django/test_django.py

```diff
@@ -49,6 +50,98 @@ def setUp(self):
             'ASAP_KEY_RETRIEVER_CLASS': self.retriever
         }
 
+
+@modify_settings(MIDDLEWARE={
+    'prepend': 'atlassian_jwt_auth.contrib.django.middleware.ASAPMiddleware',
+})
+class TestAsapMiddleware(DjangoAsapMixin, RS256KeyTestMixin, SimpleTestCase):
```

Comment:
We are missing tests for checking the `subject` where only specific `subjects` are allowed.